src/ResourceManager/AzureBatch/Commands.Batch/packages.config

```diff
@@ -1,7 +1,22 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Azure.Batch" version="5.1.2" targetFramework="net45" />
+  <package id="Azure.Batch" version="8.0.0" targetFramework="net452" />
+  <package id="Microsoft.Azure.Common" version="2.1.0" targetFramework="net45" />
+  <package id="Microsoft.Azure.Common.Dependencies" version="1.0.0" targetFramework="net45" />
   <package id="Microsoft.Azure.KeyVault.Core" version="1.0.0" targetFramework="net45" />
-  <package id="Microsoft.Azure.Management.Batch" version="3.0.0" targetFramework="net45" />
-  <package id="WindowsAzure.Storage" version="6.2.0" targetFramework="net452" />
+  <package id="Microsoft.Azure.Management.Batch" version="4.2.0" targetFramework="net452" />
+  <package id="Microsoft.Azure.Management.Resources" version="2.20.0-preview" targetFramework="net45" />
+  <package id="Microsoft.Bcl" version="1.1.9" targetFramework="net45" />
```

Comment:
The common packages are all being resrtored by the resource manager common package.  We intentionally removed this to prevent different packages taking dependencies on different versions of common libraries.  Please revart, other than the changes to batch sdk versions

Reply:
Ah, will fix -- I think I used nuget to update the package dependencies and it probably added them back.